common_utils.py

```diff
@@ -0,0 +1,132 @@
+import configparser
```

Comment:
I don't think this should be called 'common' as it is primarily used by improvecentrality and doesn't really share with the other scripts

Reply:
I moved the load_config back to improvecentrality.py

Reply:
I believe all functions in this file are only used by improvecentrality, not just the config loader. I don't mind splitting improvecentrality into multiple files if it keeps the code cleaner, but I don't want to title this file 'common' if it doesn't share anything.

Reply:
What name would you prefer?

Reply:
common_utils has been renamed bc_utils and I moved load_config back. 

There was some confusion on my part. Load config was the only function that mentions anything about improvecentrality. I figured the other functions could still be considered common.